### ISSUE TITLE
Change distro search to use grep.metacpan.org

### ISSUE THREAD

**Issue by toddr** [metacpan/metacpan-web#1886]
This search https://metacpan.org/search?q=dist%3AGit-Repository&q=sub
should become: https://grep.metacpan.org/search?size=20&q=sub&qd=Git-Repository&qft=

**Comment by toddr** [comment]
But it needs to point to the distro not the perl module
So here to search Cwd:
https://grep.metacpan.org/search?q=sub&qd=PathTools&qft=

**Comment by oalders** [comment]
I don't think that grep is a drop-in replacement for search within a distribution.  I could see adding a grep link beside the distribution search link, though.  @babsveloso, do you have any thoughts on what UI makes sense here?

**Comment by babsveloso** [comment]
I guess we could do one of both:
- add a link to grep on the sidebar
- have the 1st result be the link to grep when it makes sense (my fav)

Also, it'd be good to track this link's performance to increase its visual importance and query coverage.

See mocks for the two ideas in here:
https://drive.google.com/open?id=0B1FYkYs1yULQYkpDbHVWQmFxeWM

**Comment by toddr** [comment]
I'm worried the 2 links seem to blead together into 1 sentence.

Something Missing? Search on grep::cpan and find out why.

maybe:

SOMETHING MISSING?
Find out why
or
Search on grep::cpan

does that read better?

**Comment by toddr** [comment]
I really like the idea of tracking what people do with those 2 options. I don't know the level of effort required to do so?

**Comment by oalders** [comment]
We can look at this over the next few days if we can come to some consensus.

**Comment by oalders** [comment]
@babsveloso thanks for your 🚀 response on that other issue. I can no longer find the https://drive.google.com/open?id=0B1FYkYs1yULQYkpDbHVWQmFxeWM referenced above. Do you have that handy as well?

**Comment by babsveloso** [comment]
There you go:
https://drive.google.com/drive/folders/1N4fq0B99eXXt-eaLq0ECiSy230kSgzVk?usp=sharing
